### PR TITLE
Fix/interval booking

### PR DESCRIPTION
**Pull request by danielronnkvist**
Refactored the bookingscontroller to only fetch the current month. Also simplified the logic in the controller.

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def initialize(user)
     else
       can :read, Drink
       can :read, DrinkType
-      can [:read, :create], Booking
+      can [:read, :create, :month], Booking
```
Comment (klaseskilson) [inline app/models/ability.rb]: Isn't `month` a controller method, and not a model method (which are listed here in ability)? Or am I missing something here?

**Comment by klaseskilson** [comment]
I just have one minor question regarding `Ability`. Otherwise, great work! :beer: 

Neat rewrite of `calendarCtrl` – 20 LOC less and no loss in clarity! :beers:

File: app/models/ability.rb (hunk 1 above)
Comment (danielronnkvist) [inline app/models/ability.rb]: I'm not really sure why this works. If i remove `:month` I get a redirect to root... I thought `:month` would go under `:read` but thats not the case.

Comment (klaseskilson) [inline app/models/ability.rb]: Wow, that is weird. The same thing happens for me. :squirrel:

**Comment by klaseskilson** [comment]
:beers:

**Pull request merged**